src/crawlee/parsel_crawler/_parsel_crawler.py

```diff
@@ -1,39 +1,29 @@
 from __future__ import annotations
 
-import asyncio
-import logging
-from typing import TYPE_CHECKING, Any
+from typing import TYPE_CHECKING
 
 from parsel import Selector
-from pydantic import ValidationError
 
-from crawlee import EnqueueStrategy
-from crawlee._request import BaseRequestData
-from crawlee._utils.blocked import RETRY_CSS_SELECTORS
 from crawlee._utils.docs import docs_group
-from crawlee._utils.urls import convert_to_absolute_url, is_url_absolute
-from crawlee.basic_crawler import BasicCrawler, BasicCrawlerOptions, ContextPipeline
-from crawlee.errors import SessionError
-from crawlee.http_clients import HttpxHttpClient
-from crawlee.http_crawler import HttpCrawlingContext
-from crawlee.parsel_crawler._parsel_crawling_context import ParselCrawlingContext
+from crawlee.abstract_http_crawler import AbstractHttpCrawler, HttpCrawlerOptions
+from crawlee.parsel_crawler._parsel_parser import ParselParser
+
+from ._parsel_crawling_context import ParselCrawlingContext
 
 if TYPE_CHECKING:
-    from collections.abc import AsyncGenerator, Iterable
+    from collections.abc import AsyncGenerator
 
     from typing_extensions import Unpack
 
-    from crawlee._types import BasicCrawlingContext, EnqueueLinksKwargs
+    from crawlee.abstract_http_crawler import ParsedHttpCrawlingContext
 
 
 @docs_group('Classes')
-class ParselCrawler(BasicCrawler[ParselCrawlingContext]):
+class ParselCrawler(AbstractHttpCrawler[ParselCrawlingContext, Selector]):
```

Comment:
Could we please mention either here or in the `ParselParser` what library it uses under the hood and add a link to the documentation?

Reply:
Done